src/tasks/saas-tasks.ts

```diff
@@ -0,0 +1,38 @@
+import test, { expect } from '@playwright/test';
+import { isSaaSInstance } from '../services/ShopInfo';
+import { FixtureTypes } from '../types/FixtureTypes';
+import { Task } from '../types/Task';
+
+export const SetupInstance = test.extend<{ SetupInstance: Task } & FixtureTypes>({
```

Comment:
When is this task supposed to be used? Does it always have to be called if a SaaS instance is the target?

Reply:
Yes, because saas instances need to be setup, before they can be used. You can see that in the platform MR

Reply:
Alright, I'll refactor it into a fixture